web/index.html

```diff
@@ -3,13 +3,12 @@
   <head>
     <meta charset="utf-8" />
     <title>Image export plugin</title>
-    <link rel="stylesheet" href="index-bundle.css" />
     <link rel="stylesheet" href="assets/css/styles.css" />
     <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
   </head>
 
   <body>
     <div class="page-container" id="root"></div>
-    <script src="index-bundle.js"></script>
+    <script src="index.tsx" type="module"></script>
```

Comment:
It would make sense to document supported browsers for the plugin in the readme.
From what is see, this forces at least:
- edge 16+
- firefox 60+
- chrome 61+

see https://caniuse.com/es6-module
any other restrictions that could add constraints to browser support that I'm missing?

Reply:
It was messed up by the old build script, normally Vite takes care of it. Thanks for finding this problem! Now it should look like this in the final artefact:

```html
...
<link rel="icon" type="image/x-icon" href="./assets/favicon.7fb0b556.ico" />
    <script type="module" crossorigin src="./assets/index.e04eeab3.js"></script>
    <link rel="stylesheet" href="./assets/index.fb4e8328.css">
  </head>

  <body>
    <div class="page-container" id="root"></div>
    ...
```

Reply:
thank you, so we are still using the `type="module"` attribute, which means no edge 15 compatibility. Since this is a departure from what we officially support for LKE, I would still recommend that the plugin readme makes beowser compatibility explicit.

Reply:
Added legacy plugin to Vite that is supposed to deal with it by adding trickery like 
```
<script type="module">try{import.meta.url;import("_").catch(()=>1);}catch(e){}window.__vite_is_modern_browser=true;</script>
```
and then automatic polyfills and fall back to the legacy build with no modules.

Reply:
OK got it! thank you.